style.css

```
@@ -1 +1,263 @@
+* {
+ box-sizing: border-box;
+}
+.body {
+ background-color: #e6e2e1;
+}
 
+/*Nav Bar*/
+
+.nav-flex {
+ display: flex;
+ flex-direction: row;
+ justify-content: flex-end;
+ width: 100%;
+ height: 200px;
+ vertical-align: middle;
```

I guess that what you wanted is to center the items inside the flex box vertically, because `vertical-align` does something totally different (take a read [here](https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align)).
```suggestion
  align-items: center;
```